test/bigtest/tests/sets/view-test.js

```diff
@@ -14,10 +14,10 @@ describe('Sets', () => {
   let setsTest;
 
   describe('View', () => {
-    beforeEach(function () {
+    beforeEach(async function () {
       setsTest = this.server.create('sets');
 
-      this.visit(`/settings/oaipmh/sets/${setsTest.id}/view`);
+      await this.visit(`/settings/oai-pmh/sets/${setsTest.id}/view`);
```

Comment:
I did not find a case when await is used for this.visit. Is it really needed?

Reply:
Fixed